### ISSUE TITLE
Typos in 02_Solubility_Classification

### ISSUE THREAD

**Issue by marmatti** [Edinburgh-Chemistry-Teaching/ATCP_23_24#3]
![image](https://github.com/Edinburgh-Chemistry-Teaching/ATCP_23_24/assets/61904349/fae9f566-9287-43a7-a57e-5b161de4a229)

To get started with this tutorial, let's **import** some packages.

-----------------------------------------------
![image](https://github.com/Edinburgh-Chemistry-Teaching/ATCP_23_24/assets/61904349/fe925b33-20db-4a19-819e-7681570f611f)

Introduction- Data **Preparation** and Exploratory Data **Analysis**

--------------------------------------------------
![image](https://github.com/Edinburgh-Chemistry-Teaching/ATCP_23_24/assets/61904349/d8d780dc-83bc-4190-8bf1-9bcd6e00c246)

Maybe **molecular fingerprinting - ECFP**  in bold? I do not understand if it is a **title**

--------------------------------------------------
![image](https://github.com/Edinburgh-Chemistry-Teaching/ATCP_23_24/assets/61904349/f22c7cfa-1b33-4603-934a-bbe29014afe6)

Explore what you see in X1 **in a bit** more detail!

----------------------------------------------------
![image](https://github.com/Edinburgh-Chemistry-Teaching/ATCP_23_24/assets/61904349/02f78ee4-4579-4d03-9847-369a42c3b38c)

You can use 20% of the training data **for validation.**
OR
You can use 20% of the training data **as a** validation set.

-----------------------------------------------------
![image](https://github.com/Edinburgh-Chemistry-Teaching/ATCP_23_24/assets/61904349/474225be-626b-4b92-a55a-606f7da5cac6)

Now let's get started by defining a 5-layer deep neural network which takes **as** an input a **1024-dimension** ECFP fingerprint obtained from the corresponding SMILES string.

-------------------------------------------------------
![image](https://github.com/Edinburgh-Chemistry-Teaching/ATCP_23_24/assets/61904349/1840d1ef-76e7-447f-a230-96d2f5412a28)

**GET RID OF EXTRA PARENTHESIS AT THE END** --> print(f'Accuracy of the network on the test data: {(100 * accuracy):.2f}%')

------------------------------------------------------
![image](https://github.com/Edinburgh-Chemistry-Teaching/ATCP_23_24/assets/61904349/b16a1205-9193-45fa-8753-5b3964dc2e7b)

**No drop down solution here**, not sure whether it is deliberate.

-------------------------------------------------------
